types/react-dropzone/v2/react-dropzone-tests.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import Dropzone = require('react-dropzone');
+
+class Test extends React.Component {
+  constructor(props: any) {
+    super(props);
+  }
+
+  dz: Dropzone;
+
+  open() {
+    this.dz.open();
+  }
+
+  render() {
+    return (
+      <div>
+        <Dropzone
+          ref={(node) => { this.dz = node } }
+          onDrop={(e: any) => { e.preventDefault(); } }
+          onDropAccepted={(e: any) => { e.preventDefault(); } }
```

Comment:
I would declare a function and pass it in here for easier maintenance in the future.